### PR TITLE
Code reviews

### PR DESCRIPTION
**Pull request by websterbontilao**

### CODE REVIEW

File: src/app/components/App.tsx [hunk 1]
```diff
@@ -1,57 +1,56 @@
...
-    const addMarker = (newMarker: any) => {
+  const addMarker = (newMarker: L.LatLng) => {
 
-        setMarkers([...markers, newMarker])
-    }
+    setMarkers([...markers, newMarker])
+  }
+
+  return (
+    <>
```
Comment (ivanrodgerapos) [inline src/app/components/App.tsx:36]: Please don't wrap with code fragments unncesessarily

### Bad ❌ 
```jsx
return ( 
  <>
    <Component>Content</Component>
  </>
)
```

### Good ✅ 
```jsx
return ( 
  <Component>Content</Component>
)
```

File: src/app/components/RoutingMachine.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import L from 'leaflet'
-import { createControlComponent } from "@react-leaflet/core"
-import 'leaflet-routing-machine'
+declare var L:any;
```
Comment (ivanrodgerapos) [inline src/app/components/RoutingMachine.ts:1]: Please check if this is necessary

File: src/app/components/RoutingMachine.ts [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
+import 'leaflet';
```
Comment (ivanrodgerapos) [inline src/app/components/RoutingMachine.ts:2]:
```suggestion
import L from 'leaflet';
```

Comment (websterbontilao) [reply src/app/components/RoutingMachine.ts:2]: The suggestion was the previous declaration, but the app encounters errors in Linter

**Review by justinenerio — APPROVED** [review]

File: src/app/components/RoutingMachine.ts (hunk 3 above)
Comment (ivanrodgerapos) [reply src/app/components/RoutingMachine.ts:2]: Please try adding `leaflet-routing-machine` types
https://www.npmjs.com/package/@types/leaflet-routing-machine

File: src/app/components/RoutingMachine.ts (hunk 2 above)
Comment (websterbontilao) [reply src/app/components/RoutingMachine.ts]: Resolved by installing [@types/leaflet-routing-machine](https://www.npmjs.com/package/@types/leaflet-routing-machine)

**Review by ivanrodgerapos — APPROVED** [review]
Looks good!

**Pull request merged**